docs/user/dql/functions.rst

```diff
@@ -1715,6 +1716,15 @@ Example::
     +-----------------------------+
 
 
+    os> SELECT MONTH_OF_YEAR(DATE('2020-08-26'))
+    fetched rows / total rows = 1/1
+    +-------------------------------------+
+    | MONTH_OF_YEAR(DATE('2020-08-26'))   |
+    |-------------------------------------|
+    | 8                                   |
+    +-------------------------------------+
+
+
```

Comment:
What happens with invalid dates (eg. 13th month, 31st day in February etc).

Reply:
It throws a semanticCheckException. Should this be added to the documentation? Or was this just a question out of curiosity?